package.json

```diff
@@ -1,15 +1,29 @@
 {
-  "scripts": {
-    "build-js": "webpack"
-  },
+  "private": true,
   "dependencies": {
-    "@babel/core": "7.12.10",
-    "@babel/preset-env": "7.12.11",
-    "@hotwired/turbo": "7.0.0-beta.1",
-    "babel-loader": "8.2.2",
-    "stimulus": "2.0.0",
-    "webpack": "5.11.1",
-    "webpack-cli": "4.3.0"
+    "@hotwired/turbo": "^7.0.0-beta.4",
+    "stimulus": "2.0.0"
   },
-  "private": true
+  "devDependencies": {
+    "@11ty/eleventy": "^0.11.1",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^3.0.6",
+    "del-cli": "^3.0.1",
+    "esbuild": "^0.8.39",
+    "markdown-it-anchor": "^7.0.1",
+    "markdown-it-toc-done-right": "^4.2.0",
+    "node-sass": "^5.0.0",
+    "npm-run-all": "^4.1.5",
+    "onchange": "^7.1.0"
+  },
+  "scripts": {
+    "serve": "npm-run-all -s clean build:js build:css -p watch:** --silent",
+    "publish": "run-s build:**",
+    "clean": "del-cli \"_public/**\"",
+    "build:11ty": "npx eleventy",
```

Comment:
Any reason not to make `eleventy` a `devDependency`? I believe `npx` will use whatever version you have installed (if any) or download the latest version.

Reply:
Oops, never mind, it's right there: https://github.com/hotwired/stimulus-site/blob/c6532e7d9ddfba7462c21f58911a55e0fd634ba0/package.json#L7-L8
I don't think you need to call it via `npx` though.

Reply:
Oops, you're correct. Don't need the `npx` prefix. Fixed.